benches/Cargo.toml

```diff
@@ -49,7 +44,3 @@ harness = false
 [[bench]]
 name = "uniform_float"
 harness = false
-
-[[bench]]
-name = "weighted"
-harness = false
```

Comment:
I'm not completely sure about this: the existing `weighted` bench *only* covers `rand` code anyway. The reason I kept it in https://github.com/rust-random/rand_distr/pull/3 is because it *could* be expanded to the other `weighted` distrs.

Maybe for now we should just keep both.